packages/react-swipeable-views/src/SwipeableViews.js

```diff
@@ -744,8 +765,38 @@ class SwipeableViews extends Component {
     } = this.state;
 
     const touchEvents = disabled ? {} : {
-      onTouchStart: this.handleTouchStart,
-      onTouchEnd: this.handleTouchEnd,
+      onTouchStart: (event) => {
+        if (onTouchStart) onTouchStart(event);
+        this.handleTouchStart(event);
+      },
+      onTouchEnd: (event) => {
+        if (onTouchEnd) onTouchEnd(event);
+        this.handleTouchEnd(event);
+      },
+      onMouseDown: (event) => {
```

Comment:
Defining the handler function in the render method is affecting performances. Using a `handleTouchStart` is preferable.